### ISSUE TITLE
Installation - New version

### ISSUE THREAD

**Issue by aguisande** [tobiasmh/Alteryx-Ldap-Input-Plugin-Binaries#4]
Hi Tobias,
I tried to update the version, and I "lost" the plugin. It was working fine in both 10.1(Admin) and 10.5(User) beta.
The steps I followed:
I deleted the original DLLs.
I copied both .DLLs to the designated locations.
Unlocked both DLLs.
Restarted Alteryx -> No LDAP Tool available
Restarted PC-> No LDAP Tool available.

Any idea?

**Comment by tobiasmh** [comment]
So to confirm you got the updated version working after the unblock process?

**Comment by aguisande** [comment]
Absolutely. I did the UNBLOCK process in the Downloads directory, prior to copying it to the \bin\Plugins directories and worked like a charm on both Alteryx versions.

 

The previous unblock process was done after copying the DLLs to the \bin\Plugins directories, and because they’re “protected somehow” directories, the DLLs remained blocked so it didn’t worked.

**Comment by aguisande** [comment]
I'm attaching a couple of screenshots of how it worked.
![ldap test config-password-password](https://cloud.githubusercontent.com/assets/17551912/14085803/8fe8d6d8-f4e7-11e5-849a-a83461141561.PNG)
![ldap test results](https://cloud.githubusercontent.com/assets/17551912/14085802/8fe69256-f4e7-11e5-84fe-be450e57ad54.PNG)

**Comment by aguisande** [comment]
Tobias,

Is there any email address I can contact you directly?

Thanks

**Comment by tobiasmh** [comment]
I have emailed you.

**Comment by jdunkerley** [comment]
Hi Tobias,

Had a similar issue when I was getting my Alteryx addins to work.

Got a simple release batch file for them which does the install on mine (https://github.com/jdunkerley/AlteryxAddIns/blob/master/AlteryxAddIns/Install.bat).

Would be pretty easy to adjust to do the install your way I think.

James

**Comment by tobiasmh** [comment]
Thanks James! Much appreciated. I'll adjust and include.

**Comment by tobiasmh** [comment]
Awesome work on that C# wrapper too, it really lowers the barrier to entry which should hopefully encourage more plugin development. 

The C++ interface is a bit brutal for those without C++ experience.

**Comment by jdunkerley** [comment]
Thanks. 

Am hoping so. Got to write up the blog series on it all.

The .Net interface is pretty raw too, took a fair while to get them acting like first party tools. No way to support BrowseAnywhere yet :(.
